server/db/fill-fake-data.js

```diff
@@ -0,0 +1,277 @@
+import fs from "fs/promises";
+import path from "path";
+
+import chai from "chai";
+import chaiHttp from "chai-http";
+import dotenv from "dotenv";
+import faker from "faker";
+
+import { fetchConfigFromDB } from "../controllers/config.js";
+import app from "../index.js";
+
+import { queryPromise } from "./database.js";
+
+chai.use(chaiHttp);
+dotenv.config();
+
+const NUMBER_OF_USERS = process.argv[2] ?? 300;
+const NUMBER_OF_DUELS = process.argv[3] ?? NUMBER_OF_USERS * 10;
+const SEED = 123;
+
+if (NUMBER_OF_USERS < 2 && NUMBER_OF_DUELS > 0) {
```

Comment:
Why we must have more than 0 duels if we have less than 2 users?

Reply:
I'm not sure to understand your question here. The check is to exit if we try to have more than 0 duel with less than 2 users.

Reply:
Oh okay, I didn't understand it like that! It's all good then :)